drive/activity-v2/quickstart/build.gradle

```diff
@@ -11,7 +11,7 @@ repositories {
 }
 
 dependencies {
-    compile 'com.google.api-client:google-api-client:1.27.0'
-    compile 'com.google.oauth-client:google-oauth-client-jetty:1.27.0'
-    compile 'com.google.apis:google-api-services-driveactivity:v2-rev20181113-1.27.0'
+    compile 'com.google.api-client:google-api-client:1.25.0'
```

Comment:
Thanks for checking!  I was about to say I went back to 1.25 because there IS no version 1.27 for the drive activity client (https://repo1.maven.org/maven2/com/google/apis/google-api-services-driveactivity/).

But as I just went to look, I see the difference -- "numbered" revisions (like "rev76") only support up to 1.25, but there are also a few "dated" revisions (like "rev20190423") which do support the very latest (1.28 now).  (Happy to chat privately if this is something we should look into.)

I've set the deps back to 1.27, using the latest dated revision for driveactivity-v2, which does have the necessary updates for this change.